### DISCUSSION TITLE
Changes to leaderboard

### DISCUSSION THREAD

**Discussion by frogfrogfrogfrogfrog** [monkeytypegame/monkeytype#1049] [Ideas] [answered]
As a casual typist I'd like to be able to get onto the leaderboard myself. I know there are things like what percentile your in being worked on currently but I could say the same for other typists. Being able to type 100 wpm is absolutely nothing when the top 20 typists everyday type at least 140 wpm. 

So could you simply extend the daily leaderboard to perhaps the top 100 people? 

Also at one point you should extend the global leaderboard as well to like the top 1000.

Another change to the leaderboard is removing the "test type" since you literally pick the test type you want to see so there's no point in saving that data and since every test is gonna be the same test type.

**Comment by unrealapex** [comment]
I agree with increasing the daily leaderboard, but I think the amount of spots on the global leaderboard should be decreased to give a bit more competition.

**Reply by frogfrogfrogfrogfrog** [reply]
I feel like if you were to decrease the amount of spots on the global leaderboard they'd be dominated by insanely high scores and it'd be really hard to get on the global leaderboard. Increased competition wouldn't really help either. Since there are only a few people who type past 200 wpm they would be the ones dominating the leaderboard. If you increase the global leaderboard it would make the leaderboard more accessible to those who only type at like 180 wpm.

**Reply by unrealapex** [reply]
True

**Reply by unrealapex** [reply]
@TheeKevin   why not make everyone who wants to go on the global leaderboard do a extremely hard typing test ? make it even harder :) 
[Typing-Exercises.txt](https://github.com/Miodec/monkeytype/files/6104925/Typing-Exercises.txt)

**Reply by frogfrogfrogfrogfrog** [reply]
@UnrealApex I read your entire thing about having an insanely hard leaderboard for those who type at lightning speeds however, the problem I find with that is that many people on the global leaderboard only typed on the easy 200 english words. Meaning that they probably don't type that fast with harder stuff since they've practiced on those 200 words an incredible amount. So it's a cool idea but seems unnecessary unless it's like some unlockable to unlock the "secret leaderboard" through typing 200 words on the global leaderboard.

**Comment by Miodec** [comment, marked as answer]
The one thing I'm worried about with increasing leaderboard sizes is time to check them. I would need to optimize the leaderboard code a bit before increasing the size I think. But yeah, having more spots would be nice, and possible in the future.
